README.md

```diff
@@ -21,23 +21,6 @@ Add the plugin in your **pubspec.yaml** fie.
 
 ### Android setup
 
-Add the following to your native implementation.
-
-```kotlin
-class MainActivity : FlutterActivity() {
```

Comment:
This is an incorrect implementation.
You are removing the option to register a [custom scheme](https://sap.github.io/gigya-android-sdk/sdk-core/#providing-a-custom-account-schema
) account to the SDK, one of the core principles.